WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsMenuViewController.swift

```diff
@@ -468,6 +480,24 @@ extension InPersonPaymentsMenuViewController {
         let setUpTapToPayViewController = PaymentSettingsFlowPresentingViewController(viewModelsAndViews: viewModelsAndViews)
         let controller = WooNavigationController(rootViewController: setUpTapToPayViewController)
         controller.navigationBar.isHidden = true
+
+        navigationController?.present(controller, animated: true)
+    }
+
+    private func presentSetUpTapToPayOnIPhoneWithOnboarding() {
+        guard let siteID = stores.sessionManager.defaultStoreID else {
```

Comment:
Should we extract `stores.sessionManager.defaultStoreID` to a siteID class parameter? As we're using it across different methods

Reply:
I think it would still need to be an optional computed if we did – this is so we can respond to changes in the store.